index.js

```diff
@@ -212,12 +212,25 @@ class Request {
             }
 
             if (res.status >= 400) {
+                body = body || {};
+                if (Buffer.isBuffer(body)) {
+                    body = body.toString();
+                }
+                if (body.constructor !== Object) {
+                    // try to parse it
+                    try {
+                        body = JSON.parse(body);
+                    } catch (ee) {
+                        // not valid JSON, just include it as-is
+                        body = { detail: body };
```

Comment:
perhaps smth more unique, perhaps with `_str` suffix to prevent exploding logstash

Reply:
I intentionally left `detail` because that's what we are using in `HTTPError` messages, so this field is already being used in logs.